libs/ui/src/bmd_paper_ballot.tsx

```diff
@@ -52,9 +62,9 @@ const Header = styled.div`
   display: flex;
   flex-direction: row;
   align-items: center;
-  border-bottom: 0.2rem solid #000000;
+  border-bottom: 0.2em solid #000000;
```

Comment:
Exactly, the root font size of the two apps does, in fact, vary. 16px is the current parent element font size in BMD, so it means the couple components that are already `em` don't have to change. 

I wondered if `px` may be clearer than `em` but I don't have a preference there, so defaulted to `em` since there were already a couple instance on the page.